hack/test-end-to-end.sh

```diff
@@ -105,6 +105,9 @@ function cleanup()
 	echo
 
 	if [[ -z "${SKIP_TEARDOWN-}" ]]; then
+		echo "[INFO] Switch back to 'default' project with 'admin' user for cleanup"
```

Comment:
I guess it would just end up running `oc project` and then tell you what the current project is, and we're already in `set +e` land, so I don't think we need the `-n` check.